### PR TITLE
[IOTDB-4549] ConfigNode snapshot test

### PR DESCRIPTION
**Pull request by CRZbulabula**
Create snapshot test in cluster integration-test for ConfigNode

### CODE REVIEW

File: confignode/src/main/java/org/apache/iotdb/confignode/manager/ConsensusManager.java [hunk 1]
```diff
@@ -83,6 +84,13 @@ private void setConsensusLayer(PartitionRegionStateMachine stateMachine) throws
                 ConsensusConfig.newBuilder()
                     .setThisNode(new TEndPoint(CONF.getInternalAddress(), CONF.getConsensusPort()))
                     .setStorageDir(CONF.getConsensusDir())
+                    .setRatisConfig(
+                        RatisConfig.newBuilder()
+                            .setSnapshot(
+                                RatisConfig.Snapshot.newBuilder()
+                                    .setAutoTriggerThreshold(100)
```
Comment (Beyyes) [inline confignode/src/main/java/org/apache/iotdb/confignode/manager/ConsensusManager.java:91]: Is this ok to set the `autoTriggerThreshold` config to 100 in master branch？

Comment (CRZbulabula) [reply confignode/src/main/java/org/apache/iotdb/confignode/manager/ConsensusManager.java:91]: @SzyWilliam has fixed this on [IOTDB-4545](https://github.com/apache/iotdb/pull/7461), I'll merge that PR later~

Comment (CRZbulabula) [reply confignode/src/main/java/org/apache/iotdb/confignode/manager/ConsensusManager.java]: Fixed~

**Review by SzyWilliam — APPROVED** [review]
LGTM

**Review by Beyyes — APPROVED** [review]
LGTM

**Pull request merged**